CollAction/Startup.cs

```diff
@@ -273,6 +273,26 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                      new { controller = "Projects", action = "Find" }
                 );
 
+                routes.MapRoute("details",
+                     "Projects/{id}/{name}/details",
```

Comment:
I think ron wanted the urls slightly different: https://collaction.atlassian.net/browse/CA-388?focusedCommentId=14643&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14643